### PR TITLE
Implemented code for splitpane menu (#1wut88u)

### PR DESCRIPTION
**Pull request by azkyakhan**
Splitpane

### CODE REVIEW

**Review by dt2patel — APPROVED** [review]
looks good to me

**Comment by azkyakhan** [comment]
picklist-size-menu-desktop
menu

picklist-size-menu-mobile

spiltpane-menu

File: src/App.vue [hunk 1]
```diff
@@ -1,58 +1,31 @@
...
-          <ion-item>
-            <ion-radio slot="start" />
-            <ion-label>20 orders</ion-label>
-            <ion-note slot="end">22 items</ion-note>
-          </ion-item>
-        </ion-list>
-      </ion-content>
-    </ion-menu>
-    <ion-router-outlet id="main" />
+    <PicklistMenu content-id="main-content" />
```
Comment (adityasharma7) [inline src/App.vue:8]: I think we could move it to open orders

File: src/components/PicklistMenu.vue [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<template>
```
Comment (adityasharma7) [inline src/components/PicklistMenu.vue:1]: Improve component name

File: src/components/PicklistMenu.vue [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      <ion-toolbar>
+        <ion-title>Picklist Size</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content>
+      <ion-list>
+        <ion-radio-group value="a">
+          <ion-item>
+            <ion-radio value="a" slot="start" />  
+            <ion-label>10 orders</ion-label>
```
Comment (adityasharma7) [inline src/components/PicklistMenu.vue:13]: Add internationalisation

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (azkyakhan) [reply src/App.vue]: Done

File: src/components/PicklistMenu.vue (hunk 2 above)
Comment (azkyakhan) [reply src/components/PicklistMenu.vue]: Done

File: src/components/PicklistMenu.vue (hunk 3 above)
Comment (azkyakhan) [reply src/components/PicklistMenu.vue]: DOne

File: src/components/Menu.vue [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  <ion-menu side="start" content-id="main-content" type="overlay" :disabled="!isUserAuthenticated">
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>Broadway {{ $t("Store") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <ion-list>
+        <ion-menu-toggle auto-hide="false" v-for="(p, i) in appPages" :key="i">
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/components/Menu.vue]: Done

File: src/components/Menu.vue [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+          <ion-item
+            button
+            @click="selectedIndex = i"
+            router-direction="root"
+            :router-link="p.url"
+            class="hydrated"
+            :class="{ selected: selectedIndex === i }">
+            <ion-icon slot="start" :ios="p.iosIcon" :md="p.mdIcon" />
+            <ion-label>{{ p.title }}</ion-label>
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/components/Menu.vue]: Done

File: src/components/Menu.vue [hunk 6]
```diff
@@ -0,0 +1,135 @@
+<template>
+  <ion-menu side="start" content-id="main-content" type="overlay" :disabled="!isUserAuthenticated">
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>Broadway {{ $t("Store") }}</ion-title>
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/components/Menu.vue]: Done

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**